react/components/ProductList/Attachment.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment, FunctionComponent, useState } from 'react'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+import { IconCaretDown, IconCaretUp } from 'vtex.styleguide'
+
+import { ProductImage } from 'vtex.order-details'
+import Price from '../Payment/FormattedPrice'
+
+interface Props {
+  attachmentsInfo: Attachment[]
+  bundleInfo: Bundle[]
+}
+
+const ProductAttachment: FunctionComponent<Props & InjectedIntlProps> = ({
+  bundleInfo,
+  attachmentsInfo,
+  intl,
+}) => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <Fragment>
+      {bundleInfo &&
+        bundleInfo.map(bundleItem => (
+          <article className="bg-muted-5 pv3 ph5 br2 mv4">
+            <div className="flex justify-between">
+              {bundleItem.imageUrl && (
+                <ProductImage url={bundleItem.imageUrl} alt={bundleItem.name} />
+              )}
+              <p className="block c-on-base">{bundleItem.name}</p>
+              <div className="flex items-center">
+                <Price value={bundleItem.price} />
+                {bundleItem.attachments && bundleItem.attachments.length > 0 && (
+                  <div
+                    className="c-action-primary ml5"
+                    onClick={() => setIsOpen(!isOpen)}
+                  >
+                    {isOpen ? <IconCaretUp /> : <IconCaretDown />}
+                  </div>
+                )}
+              </div>
+            </div>
+            {bundleItem.attachments && bundleItem.attachments.length > 0 && (
+              <div hidden={!isOpen}>
```

Comment:
not sure about this `hidden`, but w/e

Reply:
I didn't understand the problem hahahah

Reply:
I thought it was necessary to handle bundle items that did not have attachments to them. Something like this:

```
items: [
              {
                id: '22',
                skuName: 'Sku serviço',
                name: 'Produto serviço Sku serviço',
                price: 1299,
                listPrice: 1500,
                isGift: false,
                quantity: 1,
                attachments: [],
                bundleItems: [
                  {
                    id: '2',
                    attachments: [],
                    name: '[TESTE QA]',
                    price: 500,
                    quantity: 1,
                    imageUrl: null,
                    measurementUnit: 'un',
                    unitMultiplier: 1,
                  },
                ],
                seller: '1',
                imageUrl:
                  'http://vtexgame1.vteximg.com.br/arquivos/ids/155359-55-55/Produto-com-servico.png?v=635812252385100000',
                detailUrl: '/produto-servico/p',
                measurementUnit: 'un',
                unitMultiplier: 1,
              },

```

Reply:
I created this from `vtexgame1` store